{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "08badc52-5e07-4259-bed3-8be71b7283a4",
   "metadata": {},
   "source": [
    "## From Biological Intelligence to Artificial Intelligence"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "06c7f4d2-3b08-415c-b6d6-891c0f6d1fe3",
   "metadata": {},
   "source": [
    "### 1 Introduction"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "944dd8c6-7d88-4b3e-815a-7d6093cecfa3",
   "metadata": {},
   "source": [
    "Previous development of D-M model for polygenic inheritance produced a model (see Binomial Genomics) that can account for independent gene configurations. But this did not account for interactions between genes and gene-environment interactions. It was proposed a hierarchical tree model (Bayesian or HMM) would be required to generate a complete model. Here we consider that an alternative to the conventional hierarchical graph systems could be explored by a deeper understanding of how biological neural networks function. Drawing on this understanding we develop fundamental principles for a new machine learning paradigm from the ground up. This would involve significant change in thinking from basic (Boolean) logic of computer science and machine learning to incorporating neuromorphic features in affordable large scale integrated circuits such as FPGAs. This will be in line with the basic principles and algorithms developed in this set of experiments."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1eff47bd-b60a-467e-88f6-0565bf280b7f",
   "metadata": {},
   "source": [
    "The aim is to find strategies for using discrete, binarized recurrent networks, such that D-M distribution replaces Bayesian networks and bit-wise recurrent processes replace HM like methods. By simplifying the process in this way, it is hoped to build a framework that is congruent with biology and could reduce costs and improve efficiency. The realization of the Dirichlet (Alpha)-Multinomial algorithm was achieved deductively based on the basic assumptions of Mendelian inheritance and Hardy-Weinberg equation. Increasing generalizations were derived to obtain the final model using Dirichlet (alpha) multinomial distribution for estimating polygenic gene configurations. "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c283137e-c5a4-476c-8371-a51cb85d3d06",
   "metadata": {},
   "source": [
    "### 2 Description of the Book"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "61dbbb9a-f5d9-4041-8adf-e1c775a0564f",
   "metadata": {},
   "source": [
    "Here we start with the higher brain networks, in which recurrent feedback processes occur (from single neurons to large networks) with capacity to generate complex information processing including mental rotations and navigational tasks from physical to social. It is the intention of this study to generate and test hypotheses that can develop a model of recurrent biological networks that can account for information processing at all levels of the biopsychosocial spectrum:"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "621d84c1-7499-483e-9e18-3788e4da1544",
   "metadata": {},
   "source": [
    "1)\t**Individual neurons** (feedback mechanisms and logic processing)\n",
    "2)\t**Cortical Modules** and feedback processes that lead to mental rotations\n",
    "3)\t**Physical navigation** and feedback processes important to navigation including processing of landmarks and emotions\n",
    "4)\tProposal of **emotions** taking the form of **abstract landmarks** for abstract navigation mixed with physical navigation\n",
    "5)\tNeed to consider both the sphere (for rotation) and the plane (for navigation), with the former having a direct impact on the latter.\n",
    "6)\tBuild the mathematical case for sphere rotation and navigation on the **complex plane** being models for **feedback processes** on the neural tree.\n",
    "7)\tJoin this feedback model using **binarized logi**c as the edges and **D-M distribution** at each level as nodes to build the hybrid machine learning model that generalizes processes in the biopsychosocial spectrum from genes to social navigation.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6b58616e-dd2e-49ea-af3b-4953e671fe7c",
   "metadata": {},
   "source": [
    "### 3 Mathematical Framework"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "46471752-8424-4a35-b95b-d84eb0683e35",
   "metadata": {},
   "source": [
    "The following mathematical constructs will be used to develop the biological insights into models that can be used in a machine learning context:"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "98087279-704c-4f07-b9ee-6b52c95f9c73",
   "metadata": {},
   "source": [
    "1)\t**Geometric Algebra (GA)** and its isomorphism with Binomial Distribution and complex mathematics\n",
    "2)\t**Riemann sphere** and its isomorphism with the complex plane and substituting GA for complex\n",
    "3)\t**Rotational Dynamics** of the Riemann sphere and isomorphism with feedback circuits\n",
    "4)\t**XOR-XNOR logic** as fundamental logic for recursion and isomorphism with **MÃ¶bius transformations** on the Riemann Sphere.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f15cb82b-ca9f-48df-b90f-da1732bb9910",
   "metadata": {},
   "source": [
    "### 4 Developing AI Code"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "64ba9004-a5b2-468f-b7a7-a98e3ef83104",
   "metadata": {},
   "source": [
    "The final stage is to develop the machine learning algorithm based on the empirical and deductive knowledge above:"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9b36de40-b948-4012-97eb-026158f91951",
   "metadata": {},
   "source": [
    "1)\tGenerate the Machine Learning Algorithm using the mathematical models as a basis (this is important, as the **biology** informs the **mathematical models** that in turn informs the **machine learning model**. This provides a logical connection between biological networks and their simulation in a machine learning paradigm).\n",
    "2)\tBuild a hybrid model using **D-M model** and **XOR XNOR recursion**:\n",
    "  \n",
    "**Stage One-** Use pure rotational dynamics to test the emergence of membrane dynamics (because membrane dynamics is independent of gene configurations and neural networks, this would be an independent test of a model derived from assumptions about polygenic function and interactions of a network).\n",
    "  \n",
    "**Stage Two-** Test the impact of the initial model using pure XOR-XNOR recursion (rather than sphere rotation) with D-M hybrid on membrane dynamics.\n",
    "  \n",
    "**Stage Three-** Further testing of the model on simple image recognition and interaction of a Chromosome with epigenetic factors.\n",
    "  \n",
    "**Stage Four-** Foundation for development of a scale invariant canonical machine learning model that could act as the Kernel for a broad machine learning paradigm in the health field. We will call this model Life Emulating Orchestrated Network (Life emulating, given that the deductive models have biological processes as their foundation and Orchestrated due the nature of the information processing that has a graph like structure reminiscent of a symphony and in honour of the Penrose-Hameroff model OrchOR).\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6684cc98-50ac-49a1-9745-026185b52ca4",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "e72f32d8-4e78-4361-b7b2-a598025c62a5",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "31ee10da-7452-4d2b-a302-c435faf1292c",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [conda env:base] *",
   "language": "python",
   "name": "conda-base-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.13.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
